protocol/bin/flow.js

```diff
@@ -50,8 +50,11 @@ function analyze (json) {
 }
 
 function figureType (type) {
+  if (!type) {
```

Comment:
the v2 flag brings in imports (cool) but now acts slightly differently (boo). this makes our script not have to deal with this too much